libs/klv/src/main/java/org/codice/alliance/libs/klv/LocationKlvProcessor.java

```diff
@@ -109,10 +135,25 @@ private void setLocationFromCornerAttribute(Metacard metacard, Attribute attribu
                 attribute,
                 postUnionGeometryOperator,
                 preUnionGeometryOperator)
-                .ifPresent(location -> metacard.setAttribute(new AttributeImpl(
-                        AttributeNameConstants.GEOGRAPHY,
-                        location)));
+                .ifPresent(location -> setAttribute(metacard, location));
+
+    }
+
+    /**
+     * Compose the pre and post geometry operators into a single operator when
+     * working with a line string.
+     */
+    private void setLocationFromFrameCenter(Metacard metacard, Attribute attribute) {
+
+        String wkt = GeometryUtility.attributeToLineString(attribute,
```

Comment:
wky should never be empty, if the attribute doesn't have any data, then wkt will be "LINESTRING EMPTY", which is valid WKT.